sound/soc/codecs/rt1308-sdw.c

```diff
@@ -510,6 +510,14 @@ static void rt1308_sdw_shutdown(struct snd_pcm_substream *substream,
 	kfree(stream);
 }
 
+static int rt1308_set_tdm_slot(struct snd_soc_dai *dai,
+			       unsigned int tx_mask, unsigned int rx_mask,
+			       int slots, int slot_width)
+{
+	dai->channels = slots;
+	return 0;
+}
+
```

Comment:
this should not be part of rt1308-sdw, this belongs in rt1308.c since it's for I2S/TDM only.